files/manifest.json

```diff
@@ -32,6 +32,16 @@
 			"UUID": "com.amansprojects.starterpack.runcommand",
 			"PropertyInspectorPath": "propertyInspector/runCommand.html"
 		},
+		{
+			"Icon": "icons/runCommand",
+			"Name": "Run command toggle",
+			"States": [ { "Image": "actionDefaultImage", "Title": "State 1", "ShowTitle": true }, { "Image": "actionDefaultImage", "Title": "State 2", "ShowTitle": true } ],
+			"Controllers": [ "Keypad" ],
+			"SupportedInMultiActions": true,
+			"Tooltip": "Run a command with 2 toggable states",
```

Comment:
Typo, "toggleable" (it's not a real word, but it should at least have an L, and it seems that "LE" is more popular)

Reply:
No I just cant spell =)